docs/threshold.md

```diff
@@ -0,0 +1,21 @@
+# Threshold Estimators
+
+Threshold Estimators are used for identifying the threshold limit above which we regard the datapoint as anomaly. 
+It is a simple Estimator that extends BaseEstimator. 
+
+Currently, the library supports `StdDevThreshold`. This takes in paramaters `min_thresh` and `std_factor`. This model 
+defines threshold as `mean + 3 * std_factor`. 
+
+
+1. Fitting the threshold model
+```python
+thresh_clf = StdDevThreshold(std_factor=1.2)
+
+thresh_clf.fit(train_data)
```

Comment:
Can you use an output from any of the autoencoders to the threshold estimators? Using `train_data` gives the wrong usage info

Reply:
Gotcha! Will fix that